.rubocop_rspec_base.yml

```diff
@@ -4,76 +4,73 @@
 # This file contains defaults for RSpec projects. Individual projects
 # can customize by inheriting this file and overriding particular settings.
 
-AccessModifierIndentation:
+Layout/AccessModifierIndentation:
```

Comment:
We generally update this file using rspec-dev that can make multiple changes at once, I'd prefer to do that and then add the extra "version bump" commit to those prs by hand.

Reply:
Checked - .rubocop_rspec_base.yml is identical across PRs.
@petergoldstein Would you please send a PR to `rspec-dev` to update it there, too?

Reply:
@pirj Done here - https://github.com/rspec/rspec-dev/pull/297